.github/workflows/chromatic.yaml

```diff
@@ -30,13 +26,36 @@ jobs:
       - run: yarn install --frozen-lockfile
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            packages:
+              - 'packages/**'
+            draft:
+              - 'draft-packages/**'
 
       # 👇 Adds Chromatic as a step in the workflow
+
+      # Run when push event and review state is approved
+      - name: Publish to Chromatic - Merge Master into branch
```

Comment:
When the PR is approved and merging into master - Run
When the PR is approved and a commit with message that includes "Merge branch 'master' into" - Run